### PR TITLE
💄 Add: Make RUT replica configurable via envvar

### PR DESCRIPTION
**Pull request by mrnicegyu11**
## Related PR/s
https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/merge_requests/1053
## Checklist
- [x] I tested and it works

### CODE REVIEW

**Review by matusdrobuliak66 — APPROVED** [review]
Cool and we setup in all deployments to 3 and in AWS master to 0 correct?

File: services/simcore/docker-compose.yml [hunk 1]
```diff
@@ -649,7 +649,7 @@ services:
       # NOTE: https://github.com/ITISFoundation/osparc-simcore/pull/4286
       # NOTE: this MUSTN'T change, or weird things might happen
       # this will stay until all legacy dynamic services are gone.
-      replicas: 1
+      replicas: ${RESOURCE_USAGE_TRACKER_REPLICA}
```
Comment (YuryHrytsuk) [inline services/simcore/docker-compose.yml:652]:
```suggestion
      replicas: ${RESOURCE_USAGE_TRACKER_REPLICAS}
```

**Comment by YuryHrytsuk** [comment]
> Cool and we setup in all deployments to 3 and in AWS master to 0 correct?

1. Why? Don't you use it for resource usage tracking?
2. Since on AWS Master this is the only Simcore Services running on Kubernetes, I would keep it up.

**Comment by YuryHrytsuk** [comment]
> > > Cool and we setup in all deployments to 3 and in AWS master to 0 correct?
> > 
> > 
> > ```
> > 1. Why? Don't you use it for resource usage tracking?
> > 
> > 2. Since on AWS Master this is the only Simcore Services running on Kubernetes, I would keep it up.
> > ```
> 
> @matusdrobuliak66 as code owner who specifically requested this please talk to @YuryHrytsuk directly, merci :--) I am flexible to put in whatever u guys decide on,

The decision is to introduce this new ENV controlling RUT's replicas in Docker Swarm [only!] and set it to `0`, so that we will run RUT only on Kubernetes.

My confusion arose from my assumption that it will scale RUT to `0` on Kubernetes also but this is not the case.

**Review by sanderegg — APPROVED** [review]

**Pull request merged**